packages/design-system-tokens/sketch/token-importer/README.md

```diff
@@ -0,0 +1,92 @@
+# token-importer
+
+## Installation
+
+- [Download](../../releases/latest/download/token-importer.sketchplugin.zip) the latest release of the plugin
+- Un-zip
+- Double-click on token-importer.sketchplugin
+
+## Development Guide
+
+_This plugin was created using `skpm`. For a detailed explanation on how things work, checkout the [skpm Readme](https://github.com/skpm/skpm/blob/master/README.md)._
+
+### Usage
+
+Install the dependencies
+
+```bash
+npm install
+```
+
+Once the installation is done, you can run some commands inside the project folder:
+
+```bash
+npm run build
+```
+
+To watch for changes:
+
+```bash
+npm run watch
+```
+
+Additionally, if you wish to run the plugin every time it is built:
+
+```bash
+npm run start
+```
```

Comment:
[question] why use `npm` commands instead of `yarn` like the rest of the directories in this repo?

Reply:
Does it have to do with using skpm?

Reply:
Yes skpm doesn't work well with yarn, ran into issues.